### PR TITLE
bugfix/authoringtool: proper _isEnabled support

### PR DESCRIPTION
**Pull request by oliverfoster**
As authoring tool puts config on every article, need to make sure when _isEnabled: false the assessment code isn't run.

https://adaptlearning.atlassian.net/browse/ABU-1109

### CODE REVIEW

**Comment by brian-learningpool** [comment]
+1

File: js/adapt-assessmentArticleExtension.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ define([
     //Extends core/js/models/articleModel.js
     var ArticleModelInitialize = ArticleModel.prototype.initialize;
     ArticleModel.prototype.initialize = function(options) {
-        if (this.get("_assessment")) {
+        if (this.get("_assessment") && this.model.get("_assessment")._isEnabled === true) {
```
Comment (hrajotia) [inline js/adapt-assessmentArticleExtension.js]: @oliverfoster - this should be 
```if (this.get("_assessment") && this.get("_assessment")._isEnabled === true) {```

**Comment by oliverfoster** [comment]
Too true, two secs.

**Comment by hrajotia** [comment]
+1

**Comment by moloko** [comment]
+1

**Pull request merged**